pkg/openstack/loadbalancer.go

```diff
@@ -605,6 +606,32 @@ func (lbaas *LbaasV2) getLoadBalancerLegacyName(_ context.Context, _ string, ser
 	return cloudprovider.DefaultLoadBalancerName(service)
 }
 
+// this function gets the listeners tags from the service annotation and   
+func (lbaas *LbaasV2) customLoadBalanceListenerTag(service *corev1.Service) ([]string, error) {
```

Comment:
I don't see this function being called anywhere, can we add WIP in the title since it's not yet complete?

Reply:
Did you read the previous comment or my text on the commit before adding yours ?

Reply:
i understand you want to help but i think it's also great to read through the messages and have a grasp of the whole idea and phase of the current work, look into previous reviewer comment, so as not to repeat thesame thing they commented instead of looking only at the code